alacritty_terminal/src/grid/mod.rs

```diff
@@ -83,6 +114,7 @@ pub struct Grid<T> {
 
     /// Number of lines.
     ///
+    /// TODO: this invariant is incorrect.
```

Comment:
How is it incorrect? We might have talked about this previously, but this should probably be captured in an issue rather than a code comment that doesn't really explain the issue.